src/accumulator.jl

```diff
@@ -12,6 +12,16 @@ counter(T::Type) = Accumulator(T,Int)
 Accumulator{T,V<:Number}(dct::Dict{T,V}) = Accumulator{T,V}(copy(dct))
 counter{T}(dct::Dict{T,Int}) = Accumulator{T,Int}(copy(dct))
 
+"""
+```
```

Comment:
The convention is four spaces for the signature code block. Other code blocks can use `s.